src/screens/Settings/constants/settingsConstants.ts

```diff
@@ -0,0 +1 @@
+export const SETTINGS_CHANGE_TEMPERATURE_SCALE = '@@SETTINGS/SETTINGS_CHANGE_TEMPERATURE_SCALE';
```

Comment:
we can call it just like this: export const CHANGE_TEMPERATURE_SCALE = '@@SETTINGS/CHANGE_TEMPERATURE_SCALE' this way we male it more declarative and short. Let's follow this pattern